cmd/buildtools/velero.go

```diff
@@ -13,43 +16,83 @@ var updateVeleroAddonCommand = &cli.Command{
 	UsageText: environmentUsageText,
 	Action: func(c *cli.Context) error {
 		logrus.Infof("updating velero addon")
-		latest, err := GetLatestGitHubTag(c.Context, "vmware-tanzu", "velero")
-		if err != nil {
-			return fmt.Errorf("failed to get latest velero release: %w", err)
+
+		// XXX remove-me.
```

Comment:
is this something that should be done now or later at some point?

Reply:
Yeah, this should be removed before merging.

Reply:
removed.